api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -20,10 +20,8 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Captor;
 import org.mockito.Mock;
-import org.pmiops.workbench.db.dao.CdrVersionDao;
-import org.pmiops.workbench.db.dao.UserDao;
-import org.pmiops.workbench.db.dao.UserService;
-import org.pmiops.workbench.db.dao.WorkspaceService;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.dao.*;
```

Comment:
Please revert (and turn off `*` imports in your IDE) per past slack discussion.

Reply:
Right, yes! This was mentioned before... I thought I'd turned it off, but will investigate,

Reply:
Reverted... sorry about that.